stories/Button/Guideline.mdx

```diff
@@ -0,0 +1,169 @@
+
+| Do                                                      | Don't                                     |
+| ------------------------------------------------------- | ----------------------------------------- |
+| Use sentence case                                       | Use icons only                            |
+| Use specific action-oriented text for CTAs              | Use punctuation or exclamation points (!) |
+| Capitalize proper nouns                                 | Write “Click here”                        |
+| Check for consistent use across the UI                  |                                           |
+| Reserve enough space for translation to other languages |                                           |
+
+## Variations
+
+The button can be modify in different way to match its usage
+
+### Variant
+
+Buttons have a set of predefined variants to use in different context
+
+<Story of={ButtonStories.DefaultButtons} />
+
+#### Primary
+
+Used for the main action. It appears once within a group of buttons (or even a screen).  
+For example, “Continue” in a form.
+
+<Story of={ButtonStories.Primary} />
+
+#### Secondary
+
+Use for the secondary action within a group of buttons.
+
+<Story of={ButtonStories.Secondary} />
+
+#### Outline
+
+Dismissive action, such as the cancel button.
+
+<Story of={ButtonStories.Outline} />
+
+#### Danger
+
+Use for critical action, typically delete actions
+
+<Story of={ButtonStories.Danger} />
+
+### Links
+
+_to add_
+
+### Size
+
+Only 2 size are available for the button component : default and inline
+
+_Usecases of the different sizes ?_
+
+<Story of={ButtonStories.ButtonSizes} />
+
+### States
+
+#### Disabled button
+
+Disabled state only if an action on the current screen can enable it (action possibility).
+
+<Story of={ButtonStories.ButtonDisabled} />
+
+## Group of Button
+
+- Primary buttons placed on the right of a group.
+- Dismissive actions placed on the left. It allows the user to return to the previous screen or step in the process.
+
+Align button groups with the container's right side by default; however, based on the use case and visual balance it can be aligned differently.
+
+<Story of={ButtonStories.SimpleForm} />
+
+## Icon on buttons
+
+Every button gets a label.\
+Icons are optional and mainly used to show how buttons are different, to aid memory and differentiation.\
+Universally understood icons work well (ie. print, close, play/pause, save).\
+Use standard icons when their use matches their meaning, or at least the user's intent.\
+Avoid creating new icons for every action, especially infrequently used ones.\
+Avoid using an icon alone in a button. If no label is provide, then use a tooltip.
+
+### Placement
+
+When use with a label, the icon is placed on the left side.\
+The icon should be (approximately) the same height as the text.  
+Seeing the icon first help users to scan the page more easily, except for few cases such as navigation (right arrow).
```

Comment:
I forgot what we meant by: "except for few cases such as navigation (right arrow)."

Reply:
Something like a next page button?
![image](https://github.com/scality/core-ui/assets/146718185/2edb0504-92c4-4e83-8893-8f109bb9417b)

Reply:
yes, but I still don't get what is has to do with the rest of the sentence :-)

Reply:
I think it says that in this case the arrow should be after the label and not before, but this is not possible in the current version of the button

Reply:
Correct, I think we can keep having an icon only on the left side.